core/src/test/java/org/elasticsearch/search/aggregations/AggregationsBinaryIT.java

```diff
@@ -41,6 +42,8 @@
 import static org.hamcrest.core.IsNull.notNullValue;
 
 @ESIntegTestCase.SuiteScopeTestCase
+@AwaitsFix(bugUrl = "needs fixing after the search request refactor. Do we need agg binary?")
```

Comment:
query_binary and agg_binary have actually now been removed anyway so this test class can also be removed